### PR TITLE
👮 If a user tries to connect to an unsupported chain an error value will be returned by useEthers.

### PR DESCRIPTION
**Pull request by KevinsBobo**
In the documentation: 
```
networks List of intended supported chain configs. If a user tries to connect to an unsupported chain an error value will be returned by useEthers.
```
https://usedapp.readthedocs.io/en/latest/core.html?highlight=config#config
But in current version no `error` is returned.
So i added the code.
And it is consistent with the version before 0.9.1. When an `error` is returned, the chainId is returned as `undefined`.

### CODE REVIEW

**Review by dmaretskyi — APPROVED** [review]

File: packages/core/src/hooks/useEthers.ts [hunk 1]
```diff
@@ -40,10 +40,19 @@ export function useEthers(): Web3Ethers {
 
   const { networks } = useConfig()
   const supportedChainIds = networks?.map((network) => network.chainId)
-  const isUnsupportedChainId = chainId && supportedChainIds && supportedChainIds.indexOf(chainId) < 0
-  const unsupportedChainIdError = new Error("Unsupported chain id: " + chainId + ". Supported chain ids are: " + supportedChainIds + ".")
+  const isUnsupportedChainId =
+    chainId && supportedChainIds && supportedChainIds.indexOf(chainId) < 0
+  const unsupportedChainIdError = new Error(
+    "Unsupported chain id: " +
```
[the remark this answers is not in the document]
Comment (KevinsBobo) [reply packages/core/src/hooks/useEthers.ts]: It's done

File: packages/core/src/hooks/useEthers.ts [hunk 2]
```diff
@@ -61,7 +70,7 @@ export function useEthers(): Web3Ethers {
       throw new Error('setError is deprecated')
     },
 
-    error: errors[errors.length - 1],
+    error: error,
```
Comment (dmaretskyi) [inline packages/core/src/hooks/useEthers.ts:73]:
```suggestion
    error,
```

**Review by dmaretskyi — APPROVED** [review]

**Comment by KevinsBobo** [comment]
Can this PR be released in version 0.12?

**Comment by dmaretskyi** [comment]
Yes, or as a patch of 0.12.x

**Pull request merged**